projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.scss

```diff
@@ -109,6 +110,49 @@
 	}
 }
 
+/**
+ * Site badge and plan info in the dropdown
+ */
+#wp-admin-bar-site-badge {
+	display: flex;
+	flex-direction: column;
+	gap: 4px;
+	min-width: 260px;
+}
+
+#wp-admin-bar-site-badge-status .ab-item,
+#wp-admin-bar-site-badge-plan .ab-item {
+	height: auto !important;
+}
+
+#wpadminbar .wp-admin-bar__site-info {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+}
+	
+#wpadminbar .wp-admin-bar__site-info-label {
+	margin-bottom: 4px;
+}
+	
+#wpadminbar .wp-admin-bar__info-badges {
+	display: flex;
+	flex-wrap: wrap;
+	gap: 4px;
+	background-color: rgba(255, 255, 255, 0.1);
+	border-radius: 4px;
+	box-sizing: border-box;
+	padding: 0 8px;
+    min-height: 24px;
```

Comment:
The format looks weird.

Reply:
Oh looks like this file uses a weird mix of spaces and tabs, I'll try to normalize.